README.rst

```diff
@@ -260,89 +260,6 @@ Alongside all that, join our developer community on Matrix:
 `#synapse-dev:matrix.org <https://matrix.to/#/#synapse-dev:matrix.org>`_, featuring real humans!
 
 
-Quick start
------------
```

Comment:
Would it make sense to link to https://matrix-org.github.io/synapse/latest/welcome_and_overview.html or something similar here?

Reply:
or to the contributing documentation. I think it's worth pointing to that prominently from the README so that people might actually follow it if they want to contribute.

Reply:
I link to https://matrix-org.github.io/synapse/latest/ above, which points to the url Shay mentions.